spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/deployer/BackingService.java

```diff
@@ -120,6 +132,22 @@ public void setRebindOnUpdate(boolean rebindOnUpdate) {
 		this.rebindOnUpdate = rebindOnUpdate;
 	}
 
+	/**
+	 * Returns the requested updated plan if any. This enables to not trigger backing service when no plan update was
```

Comment:
I felt the design rationale for having this field (i.e. to avoid triggering unnecessary update) would not be obvious to other readers of the codebase. 

I've therefore moved this comment to the "return null" statement, as to preserve consistency in lack of javadoc in this class.

Reply:
fixed in 661f7ed3e6ef425dfdd7baabd18f96f6b4d7c287